src/selectors/isSelectedFrameVisible.js

```diff
@@ -6,10 +6,12 @@ import { originalToGeneratedId, isOriginalId } from "devtools-source-map";
 import { getSelectedFrame } from "../reducers/pause";
 import { getSelectedLocation } from "../reducers/sources";
 
-function visibleSourceId(location) {
-  return isOriginalId(location.sourceId)
-    ? originalToGeneratedId(location.sourceId)
-    : location.sourceId;
+function getGeneratedId(sourceId) {
+  if (isOriginalId(sourceId)) {
+    return originalToGeneratedId(sourceId);
+  }
+
+  return sourceId;
```

Comment:
any particular reason u moved away from using the conditional operator?

Reply:
easier on my head :)